networking/aws_load_balancer_operator/adding-tls-termination-and-ddos-protection.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/add-tls-termination-and-ddos-protection.adoc
+
+:_content-type: PROCEDURE
+[id="nw-adding-tls-termination-and-ddos-protection_{context}"]
+= Adding the tls termination and ddos-protection
+
+You can route traffic for the domain example.com to pods of Service example-service and add TLS termination and DDoS protection on the load balancer.
+
+.Prerequisites
+
+.Procedure
+
+. Deploy the operator and create an instance of the resource `AWSLoadBalancerController`:
++
+[source,yaml]
+----
+kind: AWSLoadBalancerController
+group: networking.openshift.io/v1beta1 <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: auto <3>
+  ingressClass: tls-termination <4>
+----
+
+<1>
+<2> defines name of a cluster.
+<3>
+<4>
```

Comment:
The name of the default IngressClass which will be reconciled by the provisioned `aws-load-balancer-controller`.

Reply:
Addressed